lib/main.dart

```diff
@@ -62,10 +62,5 @@ void main() async {
 }
 
 Widget buildScreen(bool emailAuthenticated) {
-  return Scaffold(
-    appBar: TopBar(title: "Main"),
```

Comment:
Removed this because it was causing a wrapper around the main menu
![image](https://user-images.githubusercontent.com/4079954/193325070-2a69b6dc-b575-4fb7-8b0e-d0fdc58419d2.png)